util/uvmdvgen/common_vseq.sv.tpl

```diff
@@ -11,7 +11,9 @@ class ${name}_common_vseq extends ${name}_base_vseq;
   `uvm_object_new
 
   virtual task body();
+% if is_cip:
     run_common_vseq_wrapper(num_trans);
+% endif
```

Comment:
```suggestion
% elif has_ral:
    run_csr_vseq_wrapper(num_trans);    
% endif
```
If extending from dv lib (and not cip), it can still run RAL tests if ral is available.

Reply:
Good point, I've updated to reflect this.